.github/workflows/test-pr.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Run Tests'
```

Comment:
@F-WRunTime, please set up branch protection on `master` with the usual stuff and required checks

* `Run Tests / Code Quality Checks` 
* `Run Tests / Unit Tests`
* `Run Tests / Integration Tests`

Thank you!